src/utils/api.js

```diff
@@ -52,4 +52,15 @@ export const removeArticleVotes = async (votes, id) => {
       console.log(err);
     });
   return res.data.article.votes;
-};
\ No newline at end of file
+};
+
+export const addNewComment = async (comment, id) => {
+  console.log(comment);
+  const res = await db
+    .post(`${baseURL}/articles/${id}/comments`, comment)
+    .catch((err) => {
+      console.log(err);
+    });
+  console.log(res);
+  return res;
```

Comment:
You are getting the comment response here! Might as well use it as mentioned above.

Also, quite a few console logs you should clear up here 👍